tests/framework/extensions/nodes/ec2/ec2.go

```diff
@@ -149,3 +173,118 @@ func getSSHKeyName(sshKeyName string) string {
 	stringSlice := strings.Split(sshKeyName, ".")
 	return stringSlice[0]
 }
+
+func createNodesCommon(client *rancher.Client, numOfInstances int, hasWindows bool, windowsVersion string) (*ec2.RunInstancesInput, error) {
+	ec2Client, err := client.GetEC2Client()
+	if err != nil {
+		return nil, err
+	}
+
+	keyName := getSSHKeyName(ec2Client.Config.AWSSSHKeyName)
+	if keyName == "" {
+		return nil, fmt.Errorf("unable to parse AWSSSHKeyName configuration")
+	}
+
+	imageID := ec2Client.Config.AWSLinuxAMI
+	instanceType := ec2Client.Config.InstanceTypeLinux
+	volumeSize := ec2Client.Config.VolumeSizeLinux
+
+	// # aws ec2 describe-images --owners amazon --filters "Name=platform,Values=windows" "Name=root-device-type,Values=ebs" "Name=name,Values=Windows*2022*Containers*"
+	if hasWindows {
+		var (
+			windowsOwner            []string
+			windows2019FilterValues []string
+			windows2022FilterValues []string
+		)
+		windowsOwner = append(windowsOwner, "amazon")
+		windowsFilter := &ec2.Filter{}
+
+		switch windowsVersion {
+		case "2019":
```

Comment:
Would it make sense to have "2019" and "2022" as consts rather than having them hardcoded in the switch case?